docs/userguide/networking/get-started-overlay.md

```diff
@@ -314,11 +311,15 @@ to have external connectivity outside of their cluster.
 
 You can try starting a second network on your existing Swarm cluster using Docker Compose.
 
-1. Log into the Swarm master.
+1. Install Docker Compose. If you install on a Mac or Windows with Docker
```

Comment:
The reference to toolbox can be useful, but I'm not sure  if we should expand on that here, or leave it up to the reader how to install Docker Compose.

/cc @moxiegirl any thoughts on that?